site/en/blog/new-in-devtools-112/index.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "What's New in DevTools (Chrome 112)"
+authors:
+  - sofiayem
+date: 2023-03-08
+description: ""
+hero: 'image/NJdAV9UgKuN8AhoaPBquL7giZQo1/G5pxOHr2wECkBu8J2ujC.jpg'
+alt: 'Recorder replay customization, CSS documentation, ignore-listing irrelevant scripts, enhanced logpoint debugging, and more.'
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-112
+---
+<!-- image/NJdAV9UgKuN8AhoaPBquL7giZQo1/G5pxOHr2wECkBu8J2ujC.jpg  -->
```

Comment:
```suggestion
<!--image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gctGASDKBFTUtOQqVq2H.png  -->
```
Generate the stable release banner with Figma. We replace the header image with this when stable release roll out. See similar PR: https://github.com/GoogleChrome/developer.chrome.com/pull/5577